src/Kovan-DssSpell.t.sol

```diff
@@ -12,8 +12,9 @@ interface Hevm {
     function store(address,bytes32,bytes32) external;
 }
 
-interface MedianizerV1Abstract {
-    function authority() external view returns (address);
+interface NewChiefAbstract {
+    function live() external view returns (uint256);
+    function launch() external;
```

Comment:
Once this PR is merged, we can merge:
https://github.com/makerdao/dss-interfaces/pull/38